components/Button.js

```diff
@@ -1,17 +1,24 @@
-import { styled } from "styled-components";
+import { css, styled } from "styled-components";
 
-export default function Button({ type, onClick, buttonText }) {
+export default function Button({ type, onClick, buttonText, $isPrimary }) {
   return (
-    <StyledButton type={type} onClick={onClick}>
+    <StyledButton type={type} onClick={onClick} $isPrimary={$isPrimary}>
```

Comment:
I think it could be good to think about renaming `$isPrimary`later to `variant` or something else for the `StyledButton` component, if you want to implement more variants and not online `isPrimary`. And for the naming of the prop of the `Button` component itself you do not need the `$` I suppose (line 3). :)

Reply:
I will do this later in another US